### PR TITLE
Exclude sov conversion feesharingproxy

### PR DESCRIPTION
**Pull request by cwsnt**
This PR includes the fix for WRBTC fee withdrawal & exclude SOV from being converted when withdrawing

### CODE REVIEW

File: contracts/modules/ProtocolSettings.sol [hunk 1]
```diff
@@ -394,27 +396,43 @@ contract ProtocolSettings is State, ProtocolTokenUser, ProtocolSettingsEvents, M
...
+						feesController, // set feeSharingProxy as receiver
+						protocolAddress, // protocol as the sender
+						tempAmount, // source token amount
+						0, // reqDestToken
+						0, // minReturn
+						"" // loan data bytes
+					);
+
+					/// Will revert if disagreement found.
+					uint256 x =
```
Comment (tjcloa) [inline contracts/modules/ProtocolSettings.sol:422]:
```suggestion
```

File: tests/FeeSharingProxyTest.js [hunk 2]
```diff
@@ -937,6 +1110,174 @@ contract("FeeSharingProxy:", (accounts) => {
...
+			whitelistedConverterList = await feeSharingProxy.getWhitelistedConverterList();
+			expect(whitelistedConverterList.length).to.equal(1);
+			expect(whitelistedConverterList[0]).to.equal(liquidityPoolV1Converter.address);
+
+			await feeSharingProxy.removeWhitelistedConverterAddress(liquidityPoolV1Converter.address);
+			whitelistedConverterList = await feeSharingProxy.getWhitelistedConverterList();
+			expect(whitelistedConverterList.length).to.equal(0);
+		});
+
+		it("should not be able to withdraw fees if converters address is not an contract address", async () => {
```
Comment (tjcloa) [inline tests/FeeSharingProxyTest.js:1148]:
```suggestion
		it("should not be able to withdraw fees if converters address is not a contract address", async () => {
```

File: tests/swaps/SwapsExternal.js [hunk 3]
```diff
@@ -305,6 +305,8 @@ contract("SwapsExternal", (accounts) => {
 		});
 
 		it("Should be able to withdraw fees", async () => {
+			const maxDisagreement = new BN(wei("15", "ether")); // need to adjust to 15% max slippage for this test cases because the LP is not rebalanced
```
Comment (tjcloa) [inline tests/swaps/SwapsExternal.js:308]: can't we just mock LP to return predefined prices & disagreement?

Comment (ororopickpocket) [reply tests/swaps/SwapsExternal.js:308]: as far as i remember, the price is not depending on any liquidity pool, but is static already.

File: contracts/modules/ProtocolSettings.sol (hunk 1 above)
Comment (smbsp) [reply contracts/modules/ProtocolSettings.sol]: Removed the unused variable

File: tests/FeeSharingProxyTest.js (hunk 2 above)
Comment (smbsp) [reply tests/FeeSharingProxyTest.js]: Done

File: tests/swaps/SwapsExternal.js (hunk 3 above)
Comment (smbsp) [reply tests/swaps/SwapsExternal.js]: Changed the max slippage to 5% and rectified test descriptions

**Pull request merged**